File: hw22/Sqrts.java
/* print square roots in Java language.  R. Brown, 9/2010 */

class Sqrts {
  public static void main(String args[]) 
  {
    System.out.println("sqrt(n)");
    System.out.println("--------");
    if (args.length < 1)
    {
    	System.out.println("Error, input arguements!");
    	return;
	}
    int value = Integer.parseInt(args[0]);
    for (int n=0;  n < value;  n++)
      System.out.println(Math.sqrt(n));
    return;
  }
}

File: hw22/sqrts.c
/* print square roots in C language.  R. Brown, 9/2010 */

#include <stdio.h>
#include <math.h>
#include <stdlib.h>

int main(int argc, char **argv)
{
  if (argc < 1)
  {
  	printf("Error, input arguements next time!\n");
  	return 1;
  }
  int n;
  printf("sqrt(n)\n--------\n");
  int x = atoi(argv[1]);
  for (n = 0;  n < x;  n++)
  {
  	printf("%f\n", sqrt(n));
  }
  return 0;
}